tests/wsl/enable_systemd.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright 2012-2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Configure WSL users
```

Comment:
I like many things in this PR, but you forgot to fix the summary ;)

Rest LGTM and is approved from my side.

Reply:
Fixed :sweat_smile: